swift/ql/src/queries/Summary/SummaryStats.ql

```diff
@@ -1,5 +1,5 @@
 /**
- * @name Summary statistics
+ * @name Summary Statistics
```

Comment:
There's actually no style-guide requirement to capitalize every word in the `@name` meta-data field. See here: https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md#query-name-name.

Reply:
It does make the queries in this directory consistent, which is nice, if unimportant.

(I'll change them all to lower case if you prefer)

Reply:
Nah, I don't think it's worth running CI again for this small change.